examples/ticdc/README.md

```diff
@@ -0,0 +1,73 @@
+# A Basic TiDB cluster with monitoring
+
+> **Note:**
+>
+> This setup is for test or demo purpose only and **IS NOT** applicable for critical environment. Refer to the [Documents](https://pingcap.com/docs/stable/tidb-in-kubernetes/deploy/prerequisites/) for production setup.
+
+The following steps will create a TiDB cluster with monitoring, the monitoring data is not persisted by default.
+
+**Prerequisites**: 
+- Has TiDB operator `v1.1.0-beta.1` or higher version installed. [Doc](https://pingcap.com/docs/stable/tidb-in-kubernetes/deploy/tidb-operator/)
```

Comment:
this is not correct for this TiCDC example?

Reply:
@mikechengwei I think no need to add an example for TiCDC, it has been described clearly in the docs and the advanced example.